dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/utils/ProcessUtils.java

```diff
@@ -244,11 +291,12 @@ public static void cancelApplication(TaskExecutionContext taskExecutionContext)
                     taskExecutionContext.setAppIds(String.join(TaskConstants.COMMA, appIds));
                 }
                 if (CollectionUtils.isEmpty(appIds)) {
-                    log.info("The appId is empty");
+                    log.info("The appId is empty, so there is no need to kill yarn application.");
```

Comment:
```suggestion
                    log.info("The appId is empty");
```

We don't need this since not all shell task are yarn type.